runtime/src/bank.rs

```diff
@@ -5774,11 +5774,6 @@ impl Bank {
         }
     }
 
-    /// current stake delegations for this bank
-    pub fn cloned_stake_delegations(&self) -> HashMap<Pubkey, Delegation> {
-        self.stakes.read().unwrap().stake_delegations().clone()
-    }
-
```

Comment:
This function was only called by the bank tests, so I've moved it into the test module. Originally I only needed to change the return type (to `StakeDelegations`), but saw I could also move the fn.